Alignment/CommonAlignmentAlgorithm/plugins/SiStripBackplaneCalibration.cc

```diff
@@ -320,7 +320,7 @@ void SiStripBackplaneCalibration::endOfJob() {
   // now write 'input' tree
   const SiStripBackPlaneCorrection *input = this->getBackPlaneCorrectionInput();  // never NULL
```

Comment:
Coulnt we have this line
https://github.com/cms-sw/cmssw/blob/5eeb3ac0821d74941acee2b88faba152883f16d9/Alignment/CommonAlignmentAlgorithm/plugins/SiStripBackplaneCalibration.cc#L111
as an instance? Or a smart pointer?

Reply:
why? it's not used for writing it out

Reply:
ok, good point